sources/web/datalab/polymer/modules/drive-file-manager/drive-file-manager.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * This file contains a collection of functions that call the Google Drive APIs, and are
+ * wrapped in the ApiManager class.
+ */
+
+class DriveFile extends DatalabFile {
+}
+
+/**
+ * An Google Drive specific file manager.
+ */
+class DriveFileManager implements FileManager {
+
+  private static readonly _directoryMimeType = 'application/vnd.google-apps.folder';
+
+  private _loadPromise: Promise<any>;
+
+  constructor() {
+    this._loadPromise = GapiManager.grantScope(GapiScopes.DRIVE);
```

Comment:
I see where this was removed from drive-file-manager, but I don't see where it was added to the Gapi module.